src/metabase/models/permissions_group.clj

```diff
@@ -22,7 +22,6 @@
 
 (models/defmodel PermissionsGroup :permissions_group)
 
-
```

Comment:
Is there a reason you removed this? I think this was consistent with how most namespaces are formatted (two newlines before a new section)

Reply:
I think I auto formatted it, and it didn't understand those. I'll put it back in.